src/api/signers.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Message,
+  OnChainEvent,
+  SignerOnChainEvent,
+  fromFarcasterTime,
+} from '@farcaster/hub-nodejs'
+
+import { db } from '../db/kysely.js'
+import { log } from '../lib/logger.js'
+import { formatSigners, formatVerifications } from '../lib/utils.js'
+
+/**
+ * Insert a new signer in the database
+ * @param msg Hub event in JSON format
+ */
+export async function insertSigners(events: SignerOnChainEvent[]) {
+  const signers = formatSigners(events)
+
+  try {
+    await db
+      .insertInto('signers')
+      .values(signers)
+      .onConflict((oc) => oc.columns(['fid', 'key']).doNothing())
+      .execute()
+
+    log.debug(`SIGNERS INSERTED`)
+  } catch (error) {
+    log.error(error, 'ERROR INSERTING SIGNERS')
+  }
+}
+
+/**
+ * Delete a signer from the database
+ * @param msg Hub event in JSON format
+ */
+export async function deleteSigners(events: SignerOnChainEvent[]) {
+  try {
+    await db.transaction().execute(async (trx) => {
+      for (const event of events) {
+        const data = event.signerEventBody!
+        const key = data.key
+
+        await trx
+          .deleteFrom('signers')
```

Comment:
Wdyt about soft deleting signers like the other entities? So updating a `deletedAt` column instead of deleting the row from the db